src/include/mutex_inline.h

```diff
@@ -266,6 +292,40 @@ __wt_spin_unlock(WT_SESSION_IMPL *session, WT_SPINLOCK *t)
 
 #endif
 
+/*
+ * __wt_spin_owns --
+ *     Check whether the session owns the spinlock.
+ */
+static inline bool
+__wt_spin_owns(WT_SESSION_IMPL *session, WT_SPINLOCK *t)
```

Comment:
This is very minor but it seemed like `owned` goes better with `locked` in my head.